src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -119,6 +126,20 @@ public ProbationRecord getProbationRecord(String crn, boolean applyDocumentFilte
         return probationRecord;
     }
 
+    private Mono<Conviction> enrichConviction(String crn, Conviction conviction) {
+
+        var convictionId = Long.valueOf(conviction.getConvictionId());
+        var enrichedConvictionMono = Mono.zip(
+            offenderRestClient.getBreaches(crn, convictionId),
```

Comment:
I've just been reviewing the front end and I *think*  we can drop the breaches call for the probation record view entirely but will confirm. Sorry hadn't realised this when I wrote the ticket!

Reply:
Have just realised it did this before... I'm not sure why it can't just use the inBreach flag on the Conviction though as it only seems to toggle the breach badge on and off...

Never mind! It is redundant on this page but we've raised another ticket to remove the one place this is used  (the breach page) here https://dsdmoj.atlassian.net/browse/PIC-1442 and then another to follow up and delete here https://dsdmoj.atlassian.net/browse/PIC-1489